console/console-init/ui/src/modules/address-space/containers/AddressSpaceFilterContainer/AddressSpaceFilterContainer.tsx

```diff
@@ -0,0 +1,337 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import * as React from "react";
+import { DataToolbarChip } from "@patternfly/react-core/dist/js/experimental";
+import { SelectOption, SelectOptionObject } from "@patternfly/react-core";
+import { RETURN_ALL_ADDRESS_SPACES_FOR_NAME_OR_NAMESPACE } from "graphql-module/queries";
+import { ISearchNameOrNameSpaceAddressSpaceListResponse } from "types/ResponseTypes";
```

Comment:
Can we have a little clarity in the name of this prop 'ISearchNameOrNameSpaceAddressSpaceListResponse'

Reply:
not sure about this comment. will discuss offline about this.